packages/react-examples/.storybook/preview.js

```diff
@@ -110,41 +73,23 @@ function loadStories() {
     require.context('../src/PACKAGE_NAME', true, /\.(Example|stories)\.tsx$/),
   ];
 
-  if (packageNamePlaceholder === 'react' || packageNamePlaceholder === 'react-components') {
+  if (packageNamePlaceholder === 'react') {
     // For suite package storybooks, also show the examples of re-exported component packages.
     // preview-loader will replace REACT_ DEPS with the actual list.
     contexts.push(
       require.context('../src', true, /(REACT_DEPS|PACKAGE_NAME)\/\w+\/[\w.]+\.(Example|stories)\.(tsx|mdx)$/),
     );
   }
 
-  // @TODO
-  // - this is a temporary solution until all converged packages use new storybook configuration
-  // - after new config is in place remove this whole IF
-  if (packageNamePlaceholder === 'react-components') {
-    // include package collocated stories within react-components
-    contexts.push(require.context('../../', true, /(REACT_DEPS)\/src\/[\w./]+\.(Example|stories)\.(tsx|mdx)$/));
-  }
-
   for (const req of contexts) {
     req.keys().forEach(key => {
       generateStoriesFromExamples(key, stories, req);
     });
   }
 
   // convert stories Set to array
-  const sorted = [...stories.values()].sort((s1, s2) => {
-    const order1 = getStoryOrder(s1.default.title);
-    const order2 = getStoryOrder(s2.default.title);
-    if (order1 < order2) {
-      // the lowest order goes first
-      return -1;
-    }
-    if (order1 > order2) {
-      return 1;
-    }
-    return s1.default.title > s2.default.title ? 1 : -1;
-  });
+  const sorted = [...stories.values()];
```

Comment:
Should these be sorted alphabetically or is that automatic?
```suggestion
  const sorted = [...stories.values()].sort();
```

Reply:
AFAIR the sorting logic was there solely for react-components purpose. am i right ?

Reply:
From the history it looks like there was this sorting logic prior to react-components being added:
```suggestion
  const sorted = [...stories.values()].sort((s1, s2) => (s1.default.title > s2.default.title ? 1 : -1));
```

Reply:
updated, thx!